### PR TITLE
attic: uninit

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change

Attic is currently broken on master.
The Attic git repo has not been updated since May 2015.
Arch Linux also does not have an Attic package anymore.
Borg should be able to read Attic backups using
http://borgbackup.readthedocs.io/en/stable/usage/upgrade.html#attic-and-borg-0-xx-to-borg-1-x.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

/cc maintainer @wscott

### CODE REVIEW

**Review by wscott — APPROVED** [review]
LGTM

**Comment by joachifm** [comment]
Makes sense. I wonder if something like this warrants a changelog entry?

**Comment by dotlambda** [comment]
I added a changelog entry in the "Backward Incompatibilities" section.

File: nixos/doc/manual/release-notes/rl-1803.xml [hunk 1]
```diff
@@ -231,6 +231,14 @@ following incompatible changes:</para>
       <literal>Option "XkbRules" "base"</literal> may result in broken keyboard layout.
     </para>
   </listitem>
+  <listitem>
+    <para>
+      The <literal>attic</literal> packaged was removed. A maintained fork called
+      <link xlink:href="https://www.borgbackup.org/">Borg</link> should be used instead.
```
Comment (Mic92) [inline nixos/doc/manual/release-notes/rl-1803.xml]: Would it make sense to also add `attic = borg;` in `pkgs/top-level/aliases.nix`?

Comment (dotlambda) [reply nixos/doc/manual/release-notes/rl-1803.xml]: I don't think so. The executables have different names.

Comment (dotlambda) [reply nixos/doc/manual/release-notes/rl-1803.xml]: But I just spotted a typo: packaged instead of package. Corrected that.

**Pull request merged**